vercel/turborepo
# fix docs link checker

### Description

We've been having issues with the docs link checker.  It's not in a great state, so in this PR the goals are:
- fix the problem that's been constantly failing
- get it working to run locally (before, it only runs on PRs and will immediately fail locally if not)
- update the tooling associated with doing this (much of it is years and years out of date).

### Testing Instructions

```terminal
cd docs
pnpm check-links
```

See failing CI run: https://github.com/vercel/turborepo/actions/runs/11467995496/job/31912111207
